lib/data_pull.rb

```diff
@@ -263,4 +266,48 @@ def run(args:, config:)
       )
     end
   end
+
+  class ProfileSummary
```

Comment:
I think the file is still small beans so I'm OK leaving it as is! Splitting to files means we have to worry about either explicit or auto loading of commands when we load them. Maybe one day it will be overwhelming